jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -469,7 +469,7 @@ public JibContainerBuilder runPluginExtensions(
         // config.getExtraConfiguration().get() is of type Action, so this cast always succeeds.
         // (Note generic <T> is erased at runtime.)
         Action<T> action = (Action<T>) config.getExtraConfiguration().get();
-        extraConfig = project.getObjects().newInstance(extraConfigType.get());
+        extraConfig = project.getObjects().newInstance(extraConfigType.get(), project);
```

Comment:
I thought this was only if you wanted to inject the project into the extension block?

Reply:
I see, lets just keep the constructor with Project and document it. Having 2 options will probably confuse people?